ui/components/component-library/button-base/button-base.test.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable jest/require-top-level-describe */
+import { render } from '@testing-library/react';
+import React from 'react';
+import { BUTTON_SIZES } from './button.constants';
+import { ButtonBase } from './button-base';
+
+describe('ButtonBase', () => {
+  it('should render button element correctly', () => {
+    const { getByTestId, container } = render(
+      <ButtonBase data-testid="button-base" />,
+    );
+    expect(getByTestId('button-base')).toBeDefined();
```

Comment:
Because it's most likely that the ButtonBase will always contain children would a slightly more robust test check for the text?

```suggestion
    const { getByTestId, getByText, container } = render(
      <ButtonBase data-testid="button-base">Button base</ButtonBase>,
    );
    expect(getByText('Button base')).toBeDefined();
```